server/protocols/jmap-integration-testing/cassandra-jmap-integration-testing/src/test/resources/mailetcontainer.xml

```diff
@@ -58,9 +58,8 @@
                 <spamdHost>localhost</spamdHost>
                 <spamdPort>783</spamdPort>
             </mailet>
-            <mailet match="IsMarkedAsSpam" class="ToRecipientFolder">
```

Comment:
the mailet isMarkedAsSpam seems useless now with this new design. What about removing it? My concern is that it could make users to misuse it.

Reply:
I prefer keeping a modular solution, and don't like **at all** the **ToSpamOrInboxFolder** mailet.

However I agree that with SpamAssassin positionning storage directives for localDelivery directly, we would not need **IsMarkedAsSpam** anymore.